### PR TITLE
Introducing the Paketo Liberty Buildpack

### PR DESCRIPTION
**Pull request by kevin-ortega**
For #2111

### CODE REVIEW

**Comment by kevin-ortega** [comment]
@dmuelle Sorry to have some updates but Alasdair had some suggestions to improve the blog.   The draft branch is in sync.

**Review by rxu1 — DISMISSED** [review]
Looks good!

**Comment by kevin-ortega** [comment]
image

File: posts/2022-04-01-cloud-native-liberty-buildpack.adoc [hunk 1]
```diff
@@ -88,9 +98,15 @@ Build the app:
 [source]
 pack build myapp --env BP_JAVA_APP_SERVER=liberty
 
-Your application is now transformed into an OCI image that is ready to run almost anywhere. 
+Your application is now transformed into an OCI image!
+
+== Now what? 
+With your OCI image you can run your application locally:
```
Comment (dmuelle) [inline posts/2022-04-01-cloud-native-liberty-buildpack.adoc]:
```suggestion
With your OCI image, you can run your application locally with the `docker run` command:
```

File: posts/2022-04-01-cloud-native-liberty-buildpack.adoc [hunk 2]
```diff
@@ -88,9 +98,15 @@ Build the app:
...
 
 [source]
+docker run --rm -p 9080:9080 myapp 
+
+or deploy your application to any Kubernetes-based platform such as link:https://www.redhat.com/en/technologies/cloud-computing/openshift[Red Hat OpenShift] using an link:https://github.com/OpenLiberty/open-liberty-operator[Open Liberty operator]
```
Comment (dmuelle) [inline posts/2022-04-01-cloud-native-liberty-buildpack.adoc]:
```suggestion
or deploy your application to any Kubernetes-based platform, such as link:https://www.redhat.com/en/technologies/cloud-computing/openshift[Red Hat OpenShift], by using an link:https://github.com/OpenLiberty/open-liberty-operator[Open Liberty operator]
```

File: posts/2022-04-01-cloud-native-liberty-buildpack.adoc [hunk 3]
```diff
@@ -60,11 +60,21 @@ link:https://paketo.io/[Paketo buildpacks] transform your application source cod
...
 
-The link:https://github.com/paketo-buildpacks/liberty[Paketo Cloud Native Liberty buildpack] provides the Open Liberty runtime to a workflow that produces an Open Container Initiative (OCI) image that can run just about anywhere.
+The link:https://github.com/paketo-buildpacks/liberty[Paketo Liberty buildpack] provides the Open Liberty runtime to a workflow that produces an Open Container Initiative (OCI) image that can run just about anywhere.
+
+== Why use the Paketo Liberty buildpack?
+
+* You can build your application without creating a Dockerfile!  
+* Advanced Caching - Paketo buildpacks use built-in caching to improve performance so you can quickly rebuild your application by updating only the layers that have changed.  
+* Bill-of-Materials - Built-in software bill of materials (SBOM) support to provide insights into the contents of the application image.  
+* Minimal Application Image - Images only contain what is necessary.
```
Comment (dmuelle) [inline posts/2022-04-01-cloud-native-liberty-buildpack.adoc]:
```suggestion
* Minimal Application Image: images contain only what is necessary.
```

File: posts/2022-04-01-cloud-native-liberty-buildpack.adoc [hunk 4]
```diff
@@ -60,11 +60,21 @@ link:https://paketo.io/[Paketo buildpacks] transform your application source cod
...
+* Reproducability - Reproduces the same application image digest by re-running the build.
```
Comment (dmuelle) [inline posts/2022-04-01-cloud-native-liberty-buildpack.adoc]:
```suggestion
* Reproducibility: reproduce the same application image digest by re-running the build. 
```

File: posts/2022-04-01-cloud-native-liberty-buildpack.adoc [hunk 5]
```diff
@@ -60,11 +60,21 @@ link:https://paketo.io/[Paketo buildpacks] transform your application source cod
...
-Paketo buildpacks are toolkits and workflows for building reproducible container images from source code.
+Paketo buildpacks implement the link:https://buildpacks.io[Cloud Native Computing Foundation] buildpack specification to provide toolkits and workflows for building reproducible container images from source code.
 
-The link:https://github.com/paketo-buildpacks/liberty[Paketo Cloud Native Liberty buildpack] provides the Open Liberty runtime to a workflow that produces an Open Container Initiative (OCI) image that can run just about anywhere.
+The link:https://github.com/paketo-buildpacks/liberty[Paketo Liberty buildpack] provides the Open Liberty runtime to a workflow that produces an Open Container Initiative (OCI) image that can run just about anywhere.
+
+== Why use the Paketo Liberty buildpack?
+
+* You can build your application without creating a Dockerfile!  
+* Advanced Caching - Paketo buildpacks use built-in caching to improve performance so you can quickly rebuild your application by updating only the layers that have changed.
```
Comment (dmuelle) [inline posts/2022-04-01-cloud-native-liberty-buildpack.adoc]:
```suggestion
* Advanced Caching: Paketo buildpacks use built-in caching to improve performance so you can quickly rebuild your application by updating only the layers that have changed.  
```

File: posts/2022-04-01-cloud-native-liberty-buildpack.adoc [hunk 6]
```diff
@@ -60,11 +60,21 @@ link:https://paketo.io/[Paketo buildpacks] transform your application source cod
...
+Paketo buildpacks implement the link:https://buildpacks.io[Cloud Native Computing Foundation] buildpack specification to provide toolkits and workflows for building reproducible container images from source code.
 
-The link:https://github.com/paketo-buildpacks/liberty[Paketo Cloud Native Liberty buildpack] provides the Open Liberty runtime to a workflow that produces an Open Container Initiative (OCI) image that can run just about anywhere.
+The link:https://github.com/paketo-buildpacks/liberty[Paketo Liberty buildpack] provides the Open Liberty runtime to a workflow that produces an Open Container Initiative (OCI) image that can run just about anywhere.
+
+== Why use the Paketo Liberty buildpack?
+
+* You can build your application without creating a Dockerfile!  
+* Advanced Caching - Paketo buildpacks use built-in caching to improve performance so you can quickly rebuild your application by updating only the layers that have changed.  
+* Bill-of-Materials - Built-in software bill of materials (SBOM) support to provide insights into the contents of the application image.
```
Comment (dmuelle) [inline posts/2022-04-01-cloud-native-liberty-buildpack.adoc]:
```suggestion
* Bill-of-Materials: a built-in software bill of materials (SBOM) support provides insights into the contents of the application image.  
```

File: posts/2022-04-01-cloud-native-liberty-buildpack.adoc [hunk 7]
```diff
@@ -60,11 +60,21 @@ link:https://paketo.io/[Paketo buildpacks] transform your application source cod
...
+The link:https://github.com/paketo-buildpacks/liberty[Paketo Liberty buildpack] provides the Open Liberty runtime to a workflow that produces an Open Container Initiative (OCI) image that can run just about anywhere.
+
+== Why use the Paketo Liberty buildpack?
+
+* You can build your application without creating a Dockerfile!  
+* Advanced Caching - Paketo buildpacks use built-in caching to improve performance so you can quickly rebuild your application by updating only the layers that have changed.  
+* Bill-of-Materials - Built-in software bill of materials (SBOM) support to provide insights into the contents of the application image.  
+* Minimal Application Image - Images only contain what is necessary.
+* Reproducability - Reproduces the same application image digest by re-running the build. 
+* Auto-detection - Images are built directly from application source.
```
Comment (dmuelle) [inline posts/2022-04-01-cloud-native-liberty-buildpack.adoc]:
```suggestion
* Auto-detection: images are built directly from application source. 
```

File: posts/2022-04-01-cloud-native-liberty-buildpack.adoc [hunk 8]
```diff
@@ -60,11 +60,21 @@ link:https://paketo.io/[Paketo buildpacks] transform your application source cod
...
+
+== Why use the Paketo Liberty buildpack?
+
+* You can build your application without creating a Dockerfile!  
+* Advanced Caching - Paketo buildpacks use built-in caching to improve performance so you can quickly rebuild your application by updating only the layers that have changed.  
+* Bill-of-Materials - Built-in software bill of materials (SBOM) support to provide insights into the contents of the application image.  
+* Minimal Application Image - Images only contain what is necessary.
+* Reproducability - Reproduces the same application image digest by re-running the build. 
+* Auto-detection - Images are built directly from application source. 
+* Rebasing - Instant updates of base images by patching the OS layer of your image without rebuilding your source code.
```
Comment (dmuelle) [inline posts/2022-04-01-cloud-native-liberty-buildpack.adoc]:
```suggestion
* Rebasing: instantly update base images without rebuilding your source code by patching the OS layer of your image. 
```

**Review by dmuelle — CHANGES_REQUESTED** [review]
Just a few minor tweaks, let me know if you have any questions

**Comment by kevin-ortega** [comment]
Thanks for your suggestions.   Looks good.

**Review by dmuelle — APPROVED** [review]
LGTM thanks!

**Pull request merged**